components/patient-data/impl/src/main/java/org/phenotips/data/internal/controller/GeneListController.java

```diff
@@ -116,7 +118,7 @@
     private static final String JSON_REJECTEDGENES_KEY = "rejectedGenes";
 
     private static final List<String> STATUS_VALUES = Arrays.asList(INTERNAL_CANDIDATE_VALUE, INTERNAL_REJECTED_VALUE,
-        INTERNAL_SOLVED_VALUE);
+        INTERNAL_SOLVED_VALUE, INTERNAL_CARRIER_VALUE);
```

Comment:
But [the values are still hardcoded in that PR](https://github.com/phenotips/phenotips/pull/2216/files#diff-665baeaa42f5fff78d584ea090d111e2R61)...